packages/tokens/config.json

```diff
@@ -3,11 +3,11 @@
   "platforms": {
     "scss_colors": {
       "transformGroup": "scss",
-      "buildPath": "dist/",
+      "buildPath": "src/Colors/",
       "files": [
         {
-          "destination": "_colors.scss",
-          "format": "scss/variables",
+          "destination": "_colors.css",
+          "format": "css/variables",
```

Comment:
As we are going forward with `css`, should we change the transformGroup to `css` rather than keeping it `scss`?

Though there is no difference in the transformGroup for `css` and `scss`

<img width="552" alt="Screenshot 2023-12-29 at 3 42 25 PM" src="https://github.com/SigNoz/signoz-core-ui/assets/26198062/58cc9a31-a6aa-458c-a5c4-8192c6bf0ad3">

But this might be changed in the future.

Reply:
done